collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -240,6 +202,42 @@ public void onClick(View v) {
             }
         });
 
+        setTitle(getString(R.string.main_menu));
+
+        // must be at the beginning of any activity that can be called from an
```

Comment:
It might be used for [this](https://github.com/opendatakit/collect/blob/7351502e60d66ea5706ef6528befbd1827d1f6e2/collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java#L112) line.